src/features/virtualized-table-renderers/userCellRenderer.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+import * as React from 'react';
+import baseCellRenderer from './baseCellRenderer';
+import FormattedUser from './FormattedUser';
+import type { UserCellRendererCellData, UserCellRendererParams } from './flowTypes';
+
+const userCellRenderer = (cellRendererParams: UserCellRendererParams) =>
+    baseCellRenderer(cellRendererParams, ({ id, email, name, login }: UserCellRendererCellData) => (
+        <FormattedUser id={id} email={email || login} name={name} />
```

Comment:
Will need to use formattedUser as a method so it returns a string. The component won't set the title field of the cell